src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -363,6 +362,11 @@ def refreshCache(self):
                 numberOfCores = getattr(baggage, "numberOfCores", 1)
             loadedJob['numberOfCores'] = numberOfCores
 
+            # check if job should be submitted as highIO job
+            highIOjob = False
+            if newJob['type'] in ["Repack", "Merge", "Cleanup", "LogCollect"]:
```

Comment:
Even for large workflows reading in DQMIO should be quick. Plus you still spend some cpu combining he DQMIO input into a single small output file. So the IO, if it is there, is mostly on the storage element, not on the local disk. So not really an issue.